src/connector/src/common.rs

```diff
@@ -466,6 +466,24 @@ impl DorisCommon {
     }
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct StarrocksCommon {
```

Comment:
are we going to load from starrocks?

Reply:
Here are some parameters so that we can connect to Starrocks.

Reply:
we use the file to share some common props both in source and sink but I did not find it used in the source part.
how about making a folder and make this part of `mod.rs`

Reply:
This file contains many parts that are only used for the sink, not the source. I will create a new pull request to extract them.